Formula/arangodb.rb

```diff
@@ -71,11 +74,28 @@ def install
       end
 
       system "cmake", "..", *args
-      system "make", "V=1", "Verbose=1", "VERBOSE=1", "install"
+      system "make", "install"
+
+      %w[arangod arango-dfdb arangosh foxx-manager].each do |f|
+        inreplace etc/"arangodb3/#{f}.conf", pkgshare, opt_pkgshare
+      end
     end
   end
 
   def post_install
+    oldpath = Regexp.escape("#{HOMEBREW_PREFIX}/Cellar/arangodb/3.0.") + "[12]"
+
+    %w[arangod arango-dfdb arangosh foxx-manager].each do |f|
+      file = etc/"arangodb3/#{f}.conf"
+      oldconfig = File.read(file)
```

Comment:
`file` is going to be a `Pathname` instance here. We thus prefer to use `file.read`.